core/amber/src/main/scala/edu/uci/ics/amber/engine/architecture/messaginglayer/OutputManager.scala

```diff
@@ -87,6 +69,24 @@ class OutputManager(
     outputGateway: NetworkOutputGateway
 ) extends AmberLogging {
 
+  // create a corresponding partitioner for the given partitioning policy
+  def toPartitioner(partitioning: Partitioning): Partitioner = {
```

Comment:
move back to its original location? or is there a reason to move it?

Reply:
Fixed. `actor` was not accessible; fixed by passing it into the original function.